backend/kernelCI/settings.py

```diff
@@ -206,7 +206,7 @@ def __getitem__(self, item):
     CSRF_COOKIE_SECURE = False
     SECURE_SSL_REDIRECT = False
     SECURE_HSTS_SECONDS = 3600
-    CACHE_TIMEOUT = 0
+    CACHE_TIMEOUT = 60
```

Comment:
I think we want debug without cache as the default behaior, otherwise it get's annoying to develop since every change will take one minute to reflect